tools/docker/windows/README.md

```diff
@@ -0,0 +1,63 @@
+# Windows Development Setup Instructions
+
+MMLSpark is currently working on making the ./runme build experience work natively on windows.This page is to fill the gap in the meantime using docker.
+
+## Prerequisites
+
+1) Install [Docker for Windows](https://www.docker.com/docker-windows)
+2) Enable [Docker mounted volumes](https://rominirani.com/docker-on-windows-mounting-host-directories-d96f3f056a2c)
+3) For ssh with X-forwarding install [MobaXTerm](https://mobaxterm.mobatek.net/download.html). Other X clients will probably work as well.
+4) If you have issues with running `./runme` you might have to boost your docker memory limit to something like 5GB
+
+## Git Setup
+
+1) Fork the repository on Github so you can make PRs
+
+2) Stop your git from converting linux line endings:
+```bash
+  git config --global core.autocrlf false

```

Comment:
I think this is better to do here, or in the .gitconfig file (that we should make) having the autocrlf is a very useful feature in most projects, and its only because we are doing exotic things that we need it off. Users should not default to having it off.